Lib/types.py

```diff
@@ -233,6 +233,14 @@ def __iter__(self):
             return self.__wrapped
         return self
     __await__ = __iter__
+    def __repr__(self):
+        return '<coroutine at %s, %s, file %s, line %s, code %s>' % (
```

Comment:
We can't just use 'coroutine' here, that would make it harder to debug things. I propose to use 'coroutine-like-object', which is a bit verbose but readable.